doc/nrf/releases/release-notes-1.4.0.rst

```diff
@@ -0,0 +1,962 @@
+
+    * Renamed the sample from nRF Cloud A-GPS.
+    * Added a common interface between A-GPS and SUPL in `nRF Cloud`_.
+    * Added sending of service info after a successful connection to `nRF Cloud`_ has been made.
+
+* :ref:`asset_tracker` application:
+
+    * Added handling of sensor channel ``get`` commands received from `nRF Cloud`_.
+    * Added event handler for :ref:`lte_lc_readme` events.
+    * Added the detection feature when there is no SIM card in the slot.
+    * Added support for Bosch BSEC library 1.4.8.0.
+      This breaks compatibility with older versions of the library.
+    * Added a timestamp for sensor or cloud data, or both.
+    * Added the :option:`CONFIG_UI_LED_PWM_FREQUENCY` Kconfig option for setting the LED PWM frequency.
+
+* :ref:`gps_with_supl_support_sample` sample:
+
+    * Now the sample allows to disable the **COEX0** pin when using the external antenna to lower noise from the LNA.
+    * Updated the frequency range of the external GPS amplifier.
+    * Added an option to give GPS prioritized radio access.
+    * Added functionality that increases the GPS priority when GPS is blocked for more than a configurable amount of time.
+
+* :ref:`lwm2m_client` sample:
+
+    * Fixed an invalid Kconfig option (``CONFIG_FOTA_ERASE_PROGRESSIVELY``) that prevented progressive erase during FOTA.
+    * Added :file:`overlay-nbiot.conf` with fine-tuned CoAP/LWM2M parameters for NB-IoT networks.
+    * Fixed a bug where a FOTA socket was not closed after the download (PULL mode).
+    * Added Bootstrap procedure support to the sample.
+    * Enabled the usage of the :ref:`lib_dfu_target` library for firmware updates, which allows to update both the application and the modem firmware.
+
+* :ref:`serial_lte_modem` application:
+
+    * Added support for the MQTT username and password.
+    * Added reading of status of TCP proxy server/client when it is not started or connected yet.
+    * Added support for partial reception of RX data (in TCP/IP proxy).
+    * Added AT command to change the UART baud rate.
+    * Added data mode support for TCP/UDP proxy client/server.
+    * Added support for the HTTP client service.
+    * Added FOTA support.
+    * Added TLS server support.
+
+* :ref:`mqtt_simple_sample` sample:
+
+    * Added TLS support to the sample.
+
+* :ref:`lib_nrf_cloud` library:
+
+    * Added saving of a valid session flag to settings after all subscriptions have completed, so that the persistent session is only used when the flag is valid.
+    * Replaced ``CONFIG_CLOUD_PERSISTENT_SESSIONS`` usage with Zephyr's ``CONFIG_MQTT_CLEAN_SESSION``.
+    * MQTT client ID prefix is now configurable.
+    * Added an option to set a timeout for :c:func:`send()` (:option:`CONFIG_NRF_CLOUD_SEND_TIMEOUT`).
+
+nRF5
+====
+
+The following changes are relevant for the nRF52 and nRF53 Series.
+
+nRF5340 SoC
+-----------
+
+* Added:
+
+  * :ref:`nc_bootloader` sample
```

Comment:
Again - would be nice to have a short explanation (can usually be taken from the intro of the doc).

Reply:
Added.